README.md

```diff
@@ -269,6 +269,16 @@ A `JsonApiClient::ResultSet` object should be paginatable with both `kaminari` a
 Person.where(name: 'Jeff').all
 ```
 
+## Headers
+
+You can specify the headers to send with all requests within a block with `with_headers`.
```

Comment:
I'd like to find an implementation that preserves the ActiveRecord-like syntax for queries, but I'll keep this in mind for `jsonapi-client`. Do you think this will bite us anytime soon? Do we (meaning me) have to patch this before we move forward?